packages/angular/cli/BUILD.bazel

```diff
@@ -6,12 +6,7 @@
 load("@npm//@bazel/jasmine:index.bzl", "jasmine_node_test")
```

Comment:
Hmm. No `@external_begin / end ` block around this load from `@npm`.

@josephperrott @alan-agius4 How does this work with the sync to google3?

Reply:
Nevermind. Not needed as per https://github.com/angular/angular-cli/pull/22258#discussion_r759582051